README.md

```diff
@@ -0,0 +1,32 @@
+# feast-custom-stores-demo
```

Comment:
```suggestion
# feast-custom-online-stores-demo
```

Reply:
I was hoping to add a custom offline store of down type in this same repo. Does that make sense or should the online store be isolated?

Reply:
I think they should be separate. It's rare for somebody to contribute both.